.golangci.yaml

```diff
@@ -1,4 +1,6 @@
 issues:
+  exclude:
+    - "Error return value of .* is not checked"
```

Comment:
Yes. This linting message is nothing but annoying. If a developer decides that an  error doesn't need to be checked, she should be able to do so without annoying linting messages

Reply:
In my opinion there is no need for linting this at all. Any error can be ignored if the developer sees fit. There is no additional explanation needed to do so. Furthermore I do not think that you can "accidentally" miss an error in go. The common pattern of `if err != nil` is everywhere, so you need to actively decide to ignore an error.

Additionally this error can be removed by actively ignoring the error like `_ := MyFunc()`. This however just makes the code ugly and is explicitly annoying in `defer` statements.

So I would argue we can savely ignore this linting error message, but I will follow teams decision if I am overruled.

Reply:
Yeah, I don't need a comment on obvious lines but sometimes it isn't obvious.

> The common pattern of if err != nil is everywhere, so you need to actively decide to ignore an error.

We also have lot's of places with `w.Write` where write can fail and returns an error but we never check those.
I'd bet these were not intentional. It's just very rare that these calls fail.

I'm also in favor of `_ := DoSomething()` because if there is just a line `DoSomething()` where the function returns an error I don't see if the error was ignored on purpose.

Reply:
Okok. You overruled me. Removed it again.